packages/botonic-react/src/webchat/message-list.jsx

```diff
@@ -1,26 +1,37 @@
 import React, { useContext } from 'react'
 import { WebchatContext } from '../contexts'
+import { StyledScrollbar } from './styled-scrollbar'
 import { staticAsset } from '../utils'
 
 export const WebchatMessageList = props => {
-  const { webchatState } = useContext(WebchatContext)
-  const CustomIntro = webchatState.theme && webchatState.theme.customIntro
-  const DefaultIntro = webchatState.theme && webchatState.theme.introImage && (
+  const { webchatState, useTheme } = useContext(WebchatContext)
+  const CustomIntro = useTheme('intro.custom')
+  const introImage = useTheme('intro.image')
+  const introStyle = useTheme('intro.style')
+  const scrollbarOptions = useTheme('scrollbar')
+  const DefaultIntro = introImage && (
     <img
       style={{
         maxHeight: '50%',
-        ...(webchatState.theme.introStyle || {})
+        maxWidth: '100%',
+        ...(introStyle || {})
```

Comment:
now that the code is so short with useTheme, you could directly use useTheme('intro.style') here. Just an idea for  future code

Reply:
i'm using it some lines above in fact